.github/workflows/codecov.yaml

```diff
@@ -0,0 +1,106 @@
+        continue-on-error: true
+
+      - name: Upload Jest coverage to Codecov
+        uses: codecov/codecov-action@v5
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
+          file: ./coverage/jest/lcov.info
+          name: jest-coverage
+        if: always()
+
+      - name: Build playground with coverage
+        run: pnpm playground:build
+        env:
+          NODE_ENV: production
+          CYPRESS_COVERAGE: 'true'
+          NODE_OPTIONS: '--max-old-space-size=6144'
+          APP_ID: ${{ secrets.APP_ID_PLAYGROUND }}
+          MC_API_URL: ${{ secrets.MC_API_URL }}
+          CTP_INITIAL_PROJECT_KEY: ${{ secrets.CYPRESS_PROJECT_KEY }}
+          HOST_GCP_STAGING: ${{ secrets.HOST_GCP_STAGING }}
+
+      # https://github.com/bahmutov/cypress-gh-action-split-install/blob/ca3916d4e7240ebdc337825d2d78eb354855464b/.github/workflows/tests.yml#L23-L30
+      # https://github.com/marketplace/actions/cypress-io#custom-install
+      - name: Restoring Cypress cache
+        # restore / cache the binary ourselves on Linux
+        # see https://github.com/actions/cache
+        id: cache-cypress
+        uses: actions/cache@v4
+        with:
+          path: ~/.cache/Cypress
+          key: ${{ runner.os }}-cypress-${{ hashFiles('**/package.json') }}
+
+      - name: Installing Cypress binary
+        run: pnpm cypress install && pnpm cypress cache list
+
+      - name: Run Cypress tests with coverage
+        run: pnpm start-server-and-test 'pnpm playground:start:prod:local' http-get://127.0.0.1:3001 'pnpm percy exec -- pnpm cypress:coverage'
+        continue-on-error: true
+        env:
+          NODE_ENV: test
+          CYPRESS_COVERAGE: 'true'
+          PERCY_TOKEN: ${{ secrets.PERCY_TOKEN_E2E_PLAYGROUND }}
+          CYPRESS_CI: 'true'
+          CYPRESS_LOGIN_USER: ${{ secrets.CYPRESS_LOGIN_USER }}
+          CYPRESS_LOGIN_PASSWORD: ${{ secrets.CYPRESS_LOGIN_PASSWORD }}
+          CYPRESS_PROJECT_KEY: ${{ secrets.CYPRESS_PROJECT_KEY }}
+          HOST_GCP_STAGING: ${{ secrets.HOST_GCP_STAGING }}
+          APP_ID: ${{ secrets.APP_ID_PLAYGROUND }}
+          MC_API_URL: ${{ secrets.MC_API_URL }}
+          CTP_INITIAL_PROJECT_KEY: ${{ secrets.CYPRESS_PROJECT_KEY }}
+
+      - name: Move Cypress coverage files
+        run: |
+          mkdir -p ./coverage/cypress
+          mv ./coverage/lcov.info ./coverage/cypress/lcov.info
+          mv ./coverage/coverage-final.json ./coverage/cypress/coverage-final.json
+          mv ./coverage/coverage-summary.json ./coverage/cypress/coverage-summary.json
+          mv ./coverage/clover.xml ./coverage/cypress/clover.xml
+          mv ./coverage/lcov-report ./coverage/cypress/lcov-report
+
+      - name: Upload Cypress coverage to Codecov
+        uses: codecov/codecov-action@v5
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
+          file: ./coverage/cypress/lcov.info
+          name: cypress-coverage
+        if: always()
```

Comment:
Same as above, I think this can be done [here](https://github.com/commercetools/merchant-center-application-kit/blob/e115255feebe89379a9cf8b175540b93fa63a30f/.github/workflows/main.yml#L331) to avoid duplication.